tasks/install-from-source.yml

```diff
@@ -43,7 +43,7 @@
     chdir={{ workspace }}/ruby-{{ ruby_version }}
     creates=/usr/local/bin/ruby
   with_items:
-    - ./configure --enable-shared
+    - ./configure --enable-shared {{ ruby_configure_options }}
```

Comment:
Can you instead change this entire line to:

```
  with_items:
    - "{{ ruby_source_configure_command }}"
```

And then in the defaults, have:

```yaml
ruby_source_configure_command: `./configure --enable-shared`
```

That way you can customize the entire line (and even remove `--enable-shared` if desired).